query_processor/services/content_router.py

```
@@ -0,0 +1,113 @@
+            return {
+                'search_id': str(query_request.search_id),
+                'type': query_request.request_type,
+                'status': 'success',
+                'message': f'{query_request.request_type.title()} request processed successfully',
+                'url': query_request.video_urls[0] if query_request.video_urls else query_request.original_content,
+                'video_urls': query_request.video_urls,
+                'total_videos': query_request.total_videos
+            }
+            
+        except Exception as e:
+            error_msg = f"Failed to route {query_request.request_type} request: {str(e)}"
```

Ruff (0.12.2)

Use explicit conversion flag

Replace with conversion flag

(RUF010)